hm_1/tasks/task_02_02.py

```diff
@@ -0,0 +1,21 @@
+"""
+Given an array of size n, find the most common and the least common elements.
+The most common element is the element that appears more than n // 2 times.
+The least common element is the element that appears fewer than other.
+You may assume that the array is non-empty and the most common element
+always exist in the array.
+Example 1:
+Input: [3,2,3]
+Output: 3, 2
+Example 2:
+Input: [2,2,1,1,1,2,2]
+Output: 2, 1
+"""
+from collections import Counter
+from typing import List, Tuple
+
+
+def major_and_minor_elem(inp: List) -> Tuple[int, int]:
+    minor_elem, _ = min(Counter(inp).items(), key=lambda x: x[1])
+    major_elem, _ = max(Counter(inp).items(), key=lambda x: x[1])
```

Comment:
Use action extract variable on the `Counter(inp)`. No need to create this object twice.